src/forms/Form/Col.tsx

```diff
@@ -150,10 +166,10 @@ const Col = (props: ColPropsTypes) => {
 						value={value}
 						error={error}
 						onChange={onChange}
-						onBlur={onBlur}
+						// onBlur={onBlur}

```

Comment:
If we're commenting something out we should make sure it doesn't break anything of what we currently have. If you check the playground story you'll see some components break because we're trying to execute the onBlur. We don't need to pass it, but then we need to comment it also in those other components or add a conditional to not execute them if they're undefined. 
Here are the components that throw the error (there might be more):
![image](https://user-images.githubusercontent.com/23409284/182727322-c696afbc-b3a6-41e8-8140-61fd76d3ceb1.png)

![image](https://user-images.githubusercontent.com/23409284/182727356-531754b2-ca5f-40c9-bdac-1e9be5115aae.png)

The Image Video Document... doesn't remove the error message when selecting a media type.
![image](https://user-images.githubusercontent.com/23409284/182727439-e43368cf-e8c9-4017-a04f-606bd49204db.png)

![image](https://user-images.githubusercontent.com/23409284/182727596-7cdb0754-fab5-4892-858d-ff39da2ad2e2.png)

The Table field doesn't remove the error message when selecting a media type.
![image](https://user-images.githubusercontent.com/23409284/182727660-a03574f3-fe75-4dd2-b909-57ac5309bee0.png)

![image](https://user-images.githubusercontent.com/23409284/182727970-136b2f02-a460-4164-a5d6-927fe7793773.png)

Reply:
The errors not related to onBlur, for example with Image Video Document or Table Field, were found to come from earlier branches. They will be resolved anyway.